modules/hivemq-edge-module-postgresql/src/test/java/com/hivemq/edge/adapters/postgresql/PostgreSQLProtocolAdapterInformationTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.edge.adapters.postgresql;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PostgreSQLProtocolAdapterInformationTest {
```

Comment:
Just realized there are no integration tests.
Can you add testcontainers based integration tests please?

Reply:
We can also add some ITs with an started edge